dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/business/WorkflowAPITest.java

```diff
@@ -1095,13 +1117,268 @@ public void issue5197() throws DotDataException, IOException, DotSecurityExcepti
+     * simulating the Document Management workflow
+     */
+    @Test
+    public void validatingDocumentManagementWorkflow()
+            throws DotDataException, IOException, DotSecurityException {
+        Contentlet contentlet1 = null;
+        try {
+            final User joeContributor = APILocator.getUserAPI().loadUserById("dotcms.org.2789");
+            final User janeReviewer = APILocator.getUserAPI().loadUserById("dotcms.org.2787");
+            final User chrisPublisher = APILocator.getUserAPI().loadUserById("dotcms.org.2795");
+
+            final int editPermission =
+                    PermissionAPI.PERMISSION_READ + PermissionAPI.PERMISSION_EDIT;
+            final int publishPermission = editPermission + PermissionAPI.PERMISSION_PUBLISH;
+
+            contentType4 = insertContentType(
+                    "ValidatingDMWf" + UtilMethods.dateToHTMLDate(new Date(), "MM-dd-yyyy-HHmmss"),
+                    BaseContentType.CONTENT);
+            contentTypeStructure4 = new StructureTransformer(ContentType.class.cast(contentType4))
+                    .asStructure();
+
+            Permission p = new Permission(contentType4.getPermissionId(), contributor.getId(),
+                    editPermission, true);
+            permissionAPI.save(p, contentType4, user, true);
+
+            p = new Permission(Contentlet.class.getCanonicalName(), contentType4.getPermissionId(),
+                    contributor.getId(), editPermission, true);
+            permissionAPI.save(p, contentType4, user, true);
+
+            p = new Permission(contentType4.getPermissionId(), publisher.getId(), publishPermission,
+                    true);
+            permissionAPI.save(p, contentType4, user, true);
+
+            p = new Permission(Contentlet.class.getCanonicalName(), contentType4.getPermissionId(),
+                    publisher.getId(), publishPermission, true);
+            permissionAPI.save(p, contentType4, user, true);
+
+            //get the Document Management workflow scheme
+            WorkflowScheme ws = createDocumentManagentReplica(
+                    DOCUMENT_MANAGEMENT_WORKFLOW_NAME + UtilMethods
+                            .dateToHTMLDate(new Date(), "MM-dd-yyyy-HHmmss"));
+
+            List<String> schemes = new ArrayList<>();
+            schemes.add(ws.getId());
+            workflowAPI.saveSchemeIdsForContentType(contentType4, schemes);
+
+            /*
+             * Create test content and set it up in scheme step
+		     */
+            contentlet1 = new Contentlet();
+            contentlet1.setContentTypeId(contentType4.id());
+            contentlet1.setLanguageId(APILocator.getLanguageAPI().getDefaultLanguage().getId());
+            contentlet1.setStringProperty(FIELD_VAR_NAME,
+                    "testDocumentManagement-1" + UtilMethods
+                            .dateToHTMLDate(new Date(), "MM-dd-yyyy-HHmmss"));
```

Comment:
This pattern "MM-dd-yyyy-HHmmss", which is used all over this class can be replaced by a constant

Reply:
hi @nollymar I already did that change but for this other ticket https://github.com/dotCMS/core/issues/13521, once I create the pull you will see it